multinet/swagger/template.yaml

```
@@ -203,3 +203,5 @@ tags:
     description: Table retrieval, inspection, creation, and deletion
   - name: uploader
     description: Uploaders for various input data formats
+  - name: downloader
+    description: Downloaders   for various input data formats
```

```suggestion
    description: Downloaders for various input data formats
```

Also, "downloaders for input formats" sounds weird to me. These are just downloaders for tables and graphs in general, I think.

---

Agree, I'll change it.